mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping

```diff
@@ -112,3 +124,6 @@ CLASS net/minecraft/class_350 net/minecraft/client/gui/widget/EntryListWidget
 		METHOD set (ILjava/lang/Object;)Ljava/lang/Object;
 			ARG 1 index
 			ARG 2 entry
+	CLASS class_5403 MoveDirection
+		FIELD field_25661 UP Lnet/minecraft/class_350$class_5403;
+		FIELD field_25662 DOWN Lnet/minecraft/class_350$class_5403;
```

Comment:
These ones should be automapped

Reply:
Should but aren't if you look here: https://github.com/LambdAurora/LambdaControls/blob/1.16/fabric/src/main/java/me/lambdaurora/lambdacontrols/client/LambdaInput.java#L566

Reply:
hmm, are these enums? wonder what went wrong

Reply:
Indeed, that's weird

Reply:
Yes it's enums

Reply:
No idea if it's the actual cause, but that might be affected by `MoveDirection` having no mappings before?